chains/wardenprotocol/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://blocksync.wardenprotocol.org
+    family: blockscout
+    name: Blockscout
+    url: https://explorer.wardenprotocol.org
+blocks:
+  confirmations: 7
+  estimateBlockTime: 2
+  reorgPeriod: 5
```

Comment:
I would recommend `reorgPeriod` is higher than the `confirmations` for a better user experience

maybe `confirmations` 1, `reorgPeriod` 5-10